### PR TITLE
Update README.md files to reflect 3 projects

### PR DESCRIPTION
**Pull request by JustSlone**
#### Pull request checklist
- [x] Include a change request file using `$ yarn change`
- [x] Add high-level ROADMAP link if we have it, or remove this heading
  - Removed heading for now
- [x] Add missing "Used By" in table for web components.
- [x] Add missing "Quick Start" links.
- [x] Path to "react" package is confusing since it is called office-ui-fabric-react
  - Added links to both packages/react and packages/OUFR

Moved remaining items to #15081

#### Description of changes

The README.md here doesn't reflect the nature of this repo, 3 distinct projects working toward a larger goal.  This PR updates the main README.md to reflect that format and leaves the individual project README.md files to explain their own project.

Here's the updated [**README.md**](https://github.com/JustSlone/fluentui/blob/readme-update/README.md).

#### Focus areas to test

Read through the top-level README.md and each project's README.md and ensure it reads well and feels cohesive.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,140 +1,32 @@
...
-ReactDOM.render(<PrimaryButton>I am a button.</PrimaryButton>, document.getElementById('root'));
-```
+The Fluent UI web repository is home to 3 main projects today, the following table will help navigate them and understand their differences.
 
-### Version policy
+<!-- prettier-ignore-start -->
+|   | React | React Northstar | Web Components |
+|---| ----- | --------------- | -------------- |
+| **Overview**    | Mature, refreshing with new concepts from react-northstar. | Newer, has concepts we're iterating on. | Web Component implementation of Fluent UI. |
+| **Used By**     | Office| Teams | TODO |
```
Comment (JustSlone) [inline README.md:13]: @chrisdholt same comment as prior PR. Can you help provide the "Used By" for Web Components?

File: README.md [hunk 2]
```diff
@@ -1,140 +1,32 @@
...
+| **Read Me**     | [README.md](https://github.com/microsoft/fluentui/tree/master/packages/office-ui-fabric-react/README.md) | [README.md](https://github.com/microsoft/fluentui/tree/master/packages/fluentui/README.md) | [README.md](https://github.com/microsoft/fluentui/tree/master/packages/web-components/README.md) |
+| **Repo**        | [./packages/office-ui-fabric-react](https://github.com/microsoft/fluentui/tree/master/packages/office-ui-fabric-react) | [./packages/fluentui/react-northstar](https://github.com/microsoft/fluentui/tree/master/packages/fluentui/react-northstar) | [./packages/web-components](https://github.com/microsoft/fluentui/tree/master/packages/web-components) |
+| **Quick Start** | Quick Start | [Quick Start](https://fluentsite.z22.web.core.windows.net/quick-start) | Quick Start |
```
Comment (JustSlone) [inline README.md:16]: @chrisdholt Also, do you all have a good quick start link for @fluentui/web-components?
I found this: https://www.fast.design/docs/components/getting-started/ but nothing for @fluentui/web-components specifically.

Comment (chrisdholt) [reply README.md:16]: That's part of us trying to onboard with the documentation as it is right now - I don't know that us hosting Fluent specific documentation for the controls (even though the API's are extremely similar if not the same) makes sense on the FAST site. For the quick start though, I do think it makes sense to point to the FAST documentations for *using FAST* in and of itself. For the quick start, what are we looking for there? I can point to the best current place and we can perhaps massage the messaging a bit if need be.

File: README.md (hunk 1 above)
Comment (chrisdholt) [reply README.md:13]: Let's go with "Edge" for now

File: README.md (hunk 2 above)
Comment (JustSlone) [reply README.md:16]: It's just the basic getting starting info. 

Examples: 
- https://fluentsite.z22.web.core.windows.net/quick-start
- https://developer.microsoft.com/en-us/fluentui#/get-started/web

For web-components for now we could just say See README.md and link to your readme, or if you have another link that's fine.

File: README.md (hunk 1 above)
Comment (JustSlone) [reply README.md]: Cool, changed to Edge

File: README.md (hunk 2 above)
Comment (JustSlone) [reply README.md]: Something like this: 
`[See README.md](https://github.com/microsoft/fluentui/tree/master/packages/web-components/README.md)`

**Comment by JustSlone** [comment]
@dzearing, @chrisdholt, @levithomason this is ready for review, remaining items moved to #15081

**Review by dzearing — APPROVED** [review]

**Review by chrisdholt — APPROVED** [review]

**Pull request merged**